index.js

```diff
@@ -39,16 +62,41 @@ const getAccounts = () => {
   })
 }
 
+const getSpecialTemplateParams = (command) => {
+  let returnType = ""
+  let returnExpression = ""
+
+  const pluck = (arr, key) => arr.map(item => item[key])
```

Comment:
Let's move this helper function up to the global scope for organizational purposes.